feat: UIG-2488 - vl-map-layer-style - cache feature styles

ksvlaanderen opened the pull request:
Cachen van feature styles voor performance boost
Typen van vl-map-layer-style & vl-map-layer-circle-style

Goldflow reviewed (changes requested):
ziet er goed uit
eventueel mogelijk 1 minimale test toe te voegen waarin je test dat je nieuwe caching mechanisme werkt?

als je denkt dat dit teveel tijd inneemt dan hoeft dit niet en zal ik het approven

Done!

ksvlaanderen committed:
feat: UIG-2488 - vl-map-layer-style - cache feature styles

Cachen van feature styles voor performance boost
Typen van vl-map-layer-style & vl-map-layer-circle-style

kspeltix commented:
:tada: This PR is included in version 1.10.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.10.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
